frontend_tests/node_tests/recent_topics.js

```diff
@@ -100,6 +100,7 @@ mock_esm("../../static/js/muted_topics", {
 mock_esm("../../static/js/narrow", {
```

Comment:
This commit doesn't make sense on its own. You can squash it to the commit which actually uses the function.

Reply:
Yes, a commit with an empty function doesn't help.

Reply:
Squashed them.